src/main/java/propertyFiles/archival.properties

```
@@ -0,0 +1,31 @@
+metadatablock.name=archival
+metadatablock.displayName=Archival Metadata
+metadatablock.displayFacet=Archival Metadata
+datasetfieldtype.submitToArchivalAppraisal.title=Submit to Archival Appraisal
+datasetfieldtype.archivedFrom.title=Archived from
+datasetfieldtype.archivedFor.title=Retention period
+datasetfieldtype.holdingArchive.title=Holding Archive
+datasetfieldtype.holdingArchiveName.title=Archived at Holding Archive
+datasetfieldtype.archivedAt.title=Archived at URL
+datasetfieldtype.submitToArchivalAppraisal.description=Your assessment whether the dataset should be submitted for archival appraisal
```

Hmm, maybe an "of" here? I'm not sure.

```suggestion
datasetfieldtype.submitToArchivalAppraisal.description=Your assessment of whether the dataset should be submitted for archival appraisal
```